src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -548,8 +545,8 @@ class NetworkOPsImp final
 
     std::atomic<OperatingMode> mMode;
 
-    std::atomic <bool> mNeedNetworkLedger;
-    bool m_amendmentBlocked;
+    std::atomic <bool> needNetworkLedger_ {false};
+    std::atomic <bool> amendmentBlocked_ {false};
```

Comment:
I think `atomic<bool>` will be easier to work with.  One advantage of `atomic_flag` is that it is guaranteed to be atomic, while nothing else is.  However in practice `atomic<scalar>` is always atomic (not implemented with a mutex).  And the `atomic_flag` API is not as friendly.